ipv8/messaging/anonymization/community.py

```diff
@@ -91,67 +89,44 @@ def __init__(self):
 
         # Maximum number of seconds that a circuit should exist
         self.max_time = 10 * 60
+        # Maximum number of seconds that an introduction point should exist
+        self.max_time_ip = 24 * 60 * 60
         # Maximum number of seconds before a circuit is considered inactive (and is removed)
         self.max_time_inactive = 20
         self.max_traffic = 250 * 1024 * 1024
 
         self.max_packets_without_reply = 50
-        self.dht_lookup_interval = 30
         # Maximum number of seconds circuit creation is allowed to take. Within this time period, the unverified hop
         # of the circuit can still change in case it is unresponsive.
         # TODO: set this to ~60s next time we change the community ID, or after enough nodes are using the newer code
```

Comment:
Should we set the timeout to `60` now then?

Reply:
I prefer to fix this in a separate PR, since I'm still expecting some issues with changing this value.